.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 *.log
+data.json
```

Comment:
Looks like you accidentally committed the `data.json` file before adding this to your `.gitignore` :D

Reply:
Yes - but I think this rule is not working 🤔I had unstaged data.json, but it's still added.

Reply:
Hm, the pathspec is right, so it should be! Anyway, not a big deal, we can sync up on Slack for some tips on checking stuff into the git staging area. In the meantime, you should be able to remove it using `git rm`!